ansible/configs/ent-demo-ate-vmc-sandbox/software_vmc.yml

```diff
@@ -0,0 +1,104 @@
+- name: Configure all hosts with repositories, common files and set environment key
+  hosts:
+  - all:!windows
+  become: true
+  gather_facts: false
+  tags:
+  - step003
+  - common_tasks
+  roles:
+  - role: set-repositories
+    when: repo_method is defined
+  - role: common
+    when: install_common | bool
+  - role: bastion-lite
+    when: install_bastion | bool
+  tasks:
+  - name: Add GUID to /etc/skel/.bashrc
+    lineinfile:
+      path: "/etc/skel/.bashrc"
+      regexp: "^export GUID"
+      line: "export GUID={{ guid }}"
+
+- name: Step 003.1 - Configuring Bastion Hosts
+  hosts: bastions
+  become: true
+  roles:
+  - role: bastion-lite
+    when: install_bastion | bool
+  - role: bastion-student-user
+    when: install_student_user | bool
+  tags:
+  - step003.1
+  - bastion_tasks
+  tasks:
+  - name: Setup Student SSH Key
+    when:
+    - install_student_user | bool
+    - student_name is defined
+    block:
+    - name: Copy SSH private key to student user .ssh directory
+      copy:
+        src: "/root/.ssh/{{ ssh_provision_key_name }}.pem"
+        dest: "~{{ student_name }}/.ssh/{{ ssh_provision_key_name }}.pem"
+        mode: u=rw,go=
+        owner: "{{ student_name }}"
+        remote_src: true
+
+    - name: Copy SSH public key to student user .ssh directory
+      copy:
+        src: "/root/.ssh/{{ ssh_provision_key_name }}.pub"
+        dest: "~{{ student_name }}/.ssh/{{ ssh_provision_key_name }}.pub"
+        mode: u=rw,go=
+        owner: "{{ student_name }}"
+        remote_src: true
```

Comment:
Variable `student_name` is not a good choice. Better to use something like `lab_user` or `demo_user`. The value in this variable is often not a student.